template/Makefile

```diff
@@ -22,8 +22,8 @@ remove-tmp:
 
 clean: destroy-box remove-tmp
 
-copy-consul:
-	if [ ! -f "./tmp/consul" ]; then mkdir -p ./tmp; vagrant ssh -c "cp /usr/local/bin/consul /vagrant/tmp/consul"; fi;
+example:
+	cd example; SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} vagrant up --provision
 
-test:
-	$(MAKE) -C test
+build:
+	echo "building docker image"
```

Comment:
Is it an example?

Reply:
Actually, it is a placeholder. We need to add build functionality to the template, as well as test functionality. This PR does not add that functionality, but we should add it soon